### PR TITLE
alexDcrypto added peer

### PR DESCRIPTION
**Pull request by AlexDCryptoC**
my node

### CODE REVIEW

File: CS-1001/heimdall-seeds.txt [hunk 1]
```diff
@@ -1 +1 @@
-persistent_peers = "da771d455076976e03654ef429090c1ecf16acd3@34.202.53.230:26656,50435251a37a356080f71f9faf8c70b33e4e653f@34.196.40.122:26656"
\ No newline at end of file
+persistent_peers = "da771d455076976e03654ef429090c1ecf16acd3@34.202.53.230:26656,50435251a37a356080f71f9faf8c70b33e4e653f@34.196.40.122:26656,cbd2b8a45b3c3098e157ba54e47de1c443208763@213.136.80.20:26657"
```
Comment (vaibhavchellani) [inline CS-1001/heimdall-seeds.txt]: @AlexDCryptoC I think the port here should be 26656 unless you changed somewhere in the config. Can you confirm is this is the right port?

Comment (AlexDCryptoC) [reply CS-1001/heimdall-seeds.txt]: ya you are right. The port should be 26656.

Comment (vaibhavchellani) [reply CS-1001/heimdall-seeds.txt]: We can merge this as soon as you update!

Comment (vaibhavchellani) [reply CS-1001/heimdall-seeds.txt]: Thanks!

**Review by vaibhavchellani — APPROVED** [review]
LGTM

File: CS-1001/heimdall-seeds.txt (hunk 1 above)
Comment (AlexDCryptoC) [reply CS-1001/heimdall-seeds.txt]: Perfect, thank you as well!^^

**Pull request merged**